File: Ch.01_Practice/boj2884.py
def time(H,M):
    if M >= 45:
        M = M-45
    else:
        M = (M-45) % 60 # 왜 60 나머지? 만약 20이면 (-25)%60인데 양수로 되나?
        H = (H-1) % 24
    return H,M

HV, MV= map(int, input().split()) #map의 기능은?
H, M = time(HV, MV)
print(H,M)

File: Ch.01_Practice/boj2884.cpp
#include <iostream>
#include <string>
#include <vector>

using namespace std;

int time(int h, int m);

void main()
{
    int inputa, inputb;
    printf("%d",time(inputa, inputb));

}

int time(int h, int m)
{
    if(m>=45)
    {
        m = m-45;
    }
    else
    {
        h = h - 1;
        m = m % 60;
    }
}